src/utils/time_tracker.py

```diff
@@ -0,0 +1,12 @@
+from galaxyutils.time_tracker import TimeTracker as OGTimeTracker
+
+
+class TimeTracker(OGTimeTracker):
+    def get_tracking_games(self):
+        return [game_id for game_id in self._running_games_dict]
+
+
+from galaxyutils.time_tracker import (  # noqa: ignore F401
+    GameNotTrackedException,
+    GamesStillBeingTrackedException,
+)
```

Comment:
probably not used now?

Reply:
`GamesStillBeingTrackedException` is not needed anymore and can be removed. However, `GameNotTrackedException` is still used in `get_game_time()`.